webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -48,13 +48,12 @@ const TrialChart: React.FC<Props> = ({
       const summ = await timeSeries({
         maxDatapoints: screen.width > 1600 ? 1500 : 1000,
         metricNames: metricNames,
-        scale: scale,
```

Comment:
**question:** sorry not familiar with this part, what's the deal with `scale` going away? just need a little bit of context (thanks)

Reply:
We removed the scale argument here: https://github.com/determined-ai/determined/pull/7278/files#diff-2d09f804d9f9f7a291c3c60bea080c0c71e5bca2c0aca1bb71fa73a358478db4L743 this was a simple cleanup that I thought was nice to take care of while making this fix.